release_notes/Service_Fabric_ReleaseNotes_81.md

```diff
@@ -0,0 +1,111 @@
+# Microsoft Azure Service Fabric 8.1  Release Notes
+
+This release includes the bug fixes and features described in this document. This release includes runtime, SDKs and Windows Server Standalone deployments to run on-premises. 
+
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 16 <br> Ubuntu 18 <br> Windows | 8.1.320.1 <br>  8.1.320.1 <br> 8.1.316.9590  |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 8.1.316.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 5.0.514  <br> 8.1.316 <br> 8.1.316 <br> 8.1.316 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.0 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 8.1 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current And Upcoming Breaking Changes](#Current-And-Upcoming-Breaking-Changes)
+* [Service Fabric Runtime](#service-fabric-runtime)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Announcements
+* Add support for Auxiliary Replica
+* **Preview** Add support for .NET 6.0 Service Fabric applications
+* Add API support application description update
+* Add periodic ping between RA and RAP to detect IPC failure and process stuck
+* Add support Probes for non containerized applications
+* Make cluster upgrade for node capacity updates impactless
```

Comment:
What does it mean by that capacity updates are impactless?

Reply:
It means with 'no disruption to running workloads'. The detailed description below lists that.